docs/src/pages.js

```diff
@@ -29,11 +29,11 @@ datagrid.children =
         { pathname: '/components/data-grid/getting-started' },
         { pathname: '/components/data-grid/columns' },
         { pathname: '/components/data-grid/rows' },
+        { pathname: '/components/data-grid/editing' },
         { pathname: '/components/data-grid/sorting' },
         { pathname: '/components/data-grid/filtering' },
         { pathname: '/components/data-grid/pagination' },
         { pathname: '/components/data-grid/selection' },
-        { pathname: '/components/data-grid/editing', title: '🚧 Editing' },
```

Comment:
Not sure if it is mentioned or not but we should change this in the core as well.